beams/beams/doctype/albatross_settings/albatross_settings.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2024, efeone and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Albatross Settings', {
+   /*
+   *  Applies a filter on `albatross_service_item` where is_stock_item'checkbox is unchecked in item doctype.
+   * Applies a filter to the `albatross_service_item` field based on the fetched Sales Types where is_time_sales
+    checkbox is checked in item doctype.
+   */
+    onload: function (frm) {
+        // Initialize the list for filtering
+        frm.albatross_service_item_sales_types = [];
```

Comment:
do not create new keys in form object. Create a new variable for operations like this